cmd/license-register.go

```diff
@@ -170,12 +170,20 @@ func mainLicenseRegister(ctx *cli.Context) error {
 	regInfo := getClusterRegInfo(getAdminInfo(aliasedURL), clusterName)
 
 	alreadyRegistered := false
-	apiKey, _, e := getSubnetCreds(alias)
-	fatalIf(probe.NewError(e), "Error in fetching subnet API Key")
-	if len(apiKey) > 0 {
-		alreadyRegistered = true
-		if len(accAPIKey) == 0 {
-			accAPIKey = apiKey
+	if len(accAPIKey) == 0 {
+		apiKey, _, e := getSubnetCreds(alias)
+		fatalIf(probe.NewError(e), "Error in fetching subnet API Key")
+		if len(apiKey) > 0 {
+			alreadyRegistered = true
+			if len(accAPIKey) == 0 {
```

Comment:
This condition is redundant as it is already checked in the first `if` condition.

Reply:
done. thanks